### ISSUE TITLE
Migration failure in fieldMetadata creation does not trigger full transaction rollback

### ISSUE THREAD

**Issue by ijreilly** [twentyhq/twenty#6099]
## Scope & Context

During field creation, if there an issue with the migration, the queryRunner executes a transaction rollback for that only rolls back the fieldMetadataRepository changes; the migration remains in "pending" state, it is not deleted from the workspace migrations.
As a result, if the migration is later successfully ran, it creates a discrepancy between the state of the db (column has been created) and the fieldMetadata content (no existing row in fieldMetadata for this field).
Instead, the migration should be deleted from the workspace migrations.

To do - check if field update and deletion are affected

## Technical inputs
Idea: WorkspaceMigrationRunnerService should be able to take a transactionManager as a parameter

**Comment by ijreilly** [comment]
we should keep this issue open right @Weiko ?

**Comment by Weiko** [comment]
Yes! Not sure if the strategy is still the right one but we still have this issue where a table structure wouldn't match its metadata due to this indeed

**Comment by FelixMalfait** [comment]
@Weiko checking again, is this still relevant? Thanks :)

**Comment by Weiko** [comment]
@FelixMalfait Good catch, we can close it indeed, thanks!
